libraries/RC_Channel/RC_Channels_VarInfo.h

```diff
@@ -86,7 +86,7 @@ const AP_Param::GroupInfo RC_Channels::var_info[] = {
     // @DisplayName: RC options
     // @Description: RC input options
     // @User: Advanced
-    // @Bitmask: 0:Ignore RC Receiver, 1:Ignore MAVLink Overrides
+    // @Bitmask: 0:Ignore RC Reciever, 1:Ignore MAVLink Overrides, 2:Ignore Receiver Failsafe
```

Comment:
The first receiver word occurrence was already correctly typed.

Reply:
fixed, thanks